src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -31,9 +31,6 @@
     <ProjectReference Include="win\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
       <Platform>x86</Platform>
     </ProjectReference>
-    <ProjectReference Include="win\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
```

Comment:
does this mean we aren't shipping an updated arm package? That would mean that this coreclr package is not going to work in UWP scenarios correct?